libipl/p10/ipl0.C

```diff
@@ -147,17 +147,34 @@ static int update_hwas_state_callback(struct pdbg_target* target, void *priv)
 		return GUARD_TGT_NOT_FOUND;
 	}
 
-	if (ipl_type() == IPL_TYPE_MPIPL && type == FRU_TYPE_CORE) {
+	if (ipl_type() == IPL_TYPE_MPIPL && (type == FRU_TYPE_CORE ||
```

Comment:
Any valid use case , which is only create FC guard records in mpipl path,?  the general direction is Host will handle associate deconfiguration rules not the BMC .

Reply:
Won't we treat FC as Core in the big core mode system @ojayanth?